homeassistant/components/asuswrt/__init__.py

```diff
@@ -1,18 +1,43 @@
 """Support for ASUSWRT devices."""
 
+import logging
+
 from homeassistant.config_entries import ConfigEntry
 from homeassistant.const import EVENT_HOMEASSISTANT_STOP, Platform
 from homeassistant.core import Event, HomeAssistant
+from homeassistant.exceptions import ConfigEntryAuthFailed
 
-from .const import DATA_ASUSWRT, DOMAIN
+from .bridge import migrate_legacy_protocols
+from .const import DATA_ASUSWRT, DOMAIN, FLOW_VERSION
 from .router import AsusWrtRouter
 
 PLATFORMS = [Platform.DEVICE_TRACKER, Platform.SENSOR]
 
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_migrate_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Migrate old entry."""
+    _LOGGER.debug("Migrating from version %s", entry.version)
+
+    if entry.version != FLOW_VERSION:
```

Comment:
Seem odd to use a shared constant for this? If the flow version is bumped it will migrate again?

Reply:
Fixed using dedicated constant.